tests/union_vector/union_vector_generated.h

```diff
@@ -249,6 +301,12 @@ struct MovieT : public flatbuffers::NativeTable {
   }
 };
 
+inline bool operator==(const MovieT &lhs, const MovieT &rhs) {
```

Comment:
Add at least one test which uses these operators?

Reply:
I have written a very basic tests. See EqualOperatorTest(). Is this sufficient?

Reply:
ah, didn't spot those.